build/valgrind/openssl.supp

```diff
@@ -10,7 +10,7 @@
    fun:_ZN2mk8libevent11poller_loopIXadL_Z9event_newEEXadL_Z10event_freeEEXadL_Z9event_addEEXadL_Z19event_base_dispatchEEEEvNS_3VarI10event_baseEEPNS0_6PollerE
    fun:_ZN2mk8libevent6Poller4loopEv
    fun:_ZN2mk7Reactor23loop_with_initial_eventESt8functionIFvvEE
-   fun:_ZN2mk23loop_with_initial_eventESt8functionIFvvEENS_3VarINS_7ReactorEEE
-   fun:_ZL31____C_A_T_C_H____T_E_S_T____418v
+   fun:_ZL31____C_A_T_C_H____T_E_S_T____440v
    fun:_ZN5Catch8runTestsERKNS_3PtrINS_6ConfigEEE
+   fun:main
```

Comment:
I needed to update the suppression file, because the location from which the error was triggered has now changed slightly and we need to inform Valgrind about that.